app/controllers/boxes_controller.rb

```diff
@@ -103,7 +103,11 @@ def load_box
 
   def load_box_samples
     samples = @box.samples.preload(:batch, :sample_identifiers)
-    samples = samples.scrambled if @box.blinded?
+    samples = if @box.blinded? && !params[:unblind] 
+      samples.sort_by{ |sample|  [ sample.uuid ] }
+    else
+      samples.sort_by{ |sample|  [ sample.batch_number , sample.concentration , sample.replicate ] }
+    end 
```

Comment:
It's better to order in SQL rather than having to do it in Ruby, so let's keep the `scrambled` scope, and fallback to `#sort_by` because of they're entity fields.

```suggestion
    samples = if @box.blinded? && !params[:unblind] 
      samples.scrambled
    else
      samples.sort_by{ |sample|  [ sample.batch_number , sample.concentration , sample.replicate ] }
    end 
```